index.html

```diff
@@ -161,11 +161,31 @@ <h2>Step 3: Run Examples</h2>
                 <p>
                     <h5>00 - Get Authenticated (Set Account ID and BaseUrl)</h5>
                     <p>
-                        The example folder <span class="badgeLabel">00 Authentication</span> contains a single method for finding and storing your <code>accountId</code>, obfuscating your password, and setting your <a href="https://docs.docusign.com/esign/guide/usage/quickstart.html#authenticate-and-get-your-base-url" target="_blank">BaseUrl</a> for subsequent API calls. 
+                        The example folder <span class="badgeLabel">00 Authentication</span> contains different authentication methods to generate an access token, used in our OAuth 2.0 authentication schemes.  We also supply a requests for finding and storing your <code>accountId</code> and <code>basePath</code> and also a request for utilzing a refresh token to retreive a new access token. Note that you need to <a href="https://developers.docusign.com/esign-rest-api/guides/authentication/obtaining-consent">obtain consent</a> on the first time requesting access via API. Click OK to proceed.
                     </p>
+
+                        <strong>For Authorization Code Grant:</strong>
+                    <p>
+                        For an Authorization Code Grant Token, begin by opening a web browser and navigating to 
+                        <code>https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature&client_id={{iKey}&redirect_uri=https://google.com</code> replacing {{iKey}} with your integration key and {{callback}} with a callback url (setup when generating the integration key) to obtain consent. After clicking OK, your browser will redirect to that link specified with a long string in the url returned for the <code>code</code> parameter. Use this use the Authoriztion Code for 01 Get Access Token to generate an Access token and a refresh token. Finally, use 02 Get login information to retreive the Account ID and Base Url.
```

Comment:
in addition to changing the folder structure I also changed the double handle bars on this link next to ikey as it's missing one on the right side